### DISCUSSION TITLE
Questions about EDC Connector usage in production environment

### DISCUSSION THREAD

**Discussion by wadahiro** [eclipse-tractusx/tractusx-edc#946] [Q&A]
Hello,

Recently, I've been investigating how to make different companies share data using the EDC Connector. Thanks to [Minimum Tractus-X Dataspace (MXD)](https://github.com/eclipse-tractusx/tutorial-resources/blob/1f473416d8464bed5335cbae9cb466e6b2ff396a/mxd/README.md), I managed to exchange data between Alice and Bob on my local k8s cluster. However, I'm not sure how to use the EDC Connector in a real production environment. Could someone help me with the following questions?

In MXD, I exchanged data by using `curl` command to call the [EDC Management API](https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api/0.1.4-SNAPSHOT). In a real production environment, what's the expected way to call the EDC Management API?

* Is it expected that EDC Connector will be embedded in applications within each company to call the EDC Management API and perform operations from the application?
* Or, is it expected that each participating company develops own UI that manages the EDC Connector independently, ensuring it's not tightly connected to applications? In this case, in the eclipse-edc repository, there is a demo Connector UI called [DataDashboard](https://github.com/eclipse-edc/DataDashboard). Is it expected that this could be used as a basis for creating a management UI in Tractus-X? Or, is there a plan to provide a reference implementation of the EDC Connector UI in Tractus-X?

I'd appreciate any insights or guidance on this.

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
